src/components/RadioButtonGroup/index.js

```diff
@@ -17,6 +19,20 @@ class RadioButtonGroup extends Component {
         super(props);
         this.errorId = uniqueId('error-message');
         this.groupNameId = props.name || uniqueId('options');
+        this.handleChange = this.handleChange.bind(this);
+        this.isMarkerActive = this.isMarkerActive.bind(this);
```

Comment:
there is not need to bind this.isMarkerActive since it not called later as callback